stdlib/3.4/enum.pyi

```diff
@@ -32,8 +32,9 @@ def unique(enumeration: _S) -> _S: ...
 if sys.version_info >= (3, 6):
     _auto_null = ...  # type: Any
 
-    class auto:
+    class auto(IntFlag):
```

Comment:
Can you add a comment explaining why we're lying here? Maybe link to the mypy issue or PR.

Reply:
As written in my comment, I don't think the comment is telling the truth but the inadvertent result of some misunderstanding. https://youtrack.jetbrains.com/issue/PY-53388/PyCharm-thinks-enumauto-needs-an-argument#focus=Comments-27-6301482.0-0

Reply:
If you think there are problems with how things are implemented, please open a new issue.

Reply:
Exists already: https://youtrack.jetbrains.com/issue/PY-53388/PyCharm-thinks-enumauto-needs-an-argument

Reply:
Please open a typeshed issue if you want typeshed people to do something about it.